src/internal/operators/filter.ts

```diff
@@ -30,6 +30,10 @@ export function filter<T>(predicate: (value: T, index: number) => boolean): Mono
  * import { fromEvent } from 'rxjs';
  * import { filter } from 'rxjs/operators';
  *
+ * const div = document.createElement('div');
+ * div.style.cssText = `width: 200px;height: 200px;background: #09c;`;
+ * document.body.appendChild(div);
+ *
  * const clicks = fromEvent(document, 'click');
```

Comment:
BTW, `find, findIndex, first` all need a div element to run the example. Or maybe we can refactor to use Typescript only, eg: 
```ts
range(1, 10)
  .pipe(filter(x => x % 2 === 1))
```